app/client/utils.js

```diff
@@ -127,7 +126,7 @@ export function* handleError(error) {
   }
 
   if (error.status === 401) {
-    yield put({ type: LOGIN_FAILURE, error: message });
+    yield put(LoginAction.loginFailure(message));
```

Comment:
A small bug fix here! `LOGIN_FAILURE` was not refactored into the new action type created in PR #186